### PR TITLE
Android language guide

### PR DESCRIPTION
**Pull request by CommanderK5**

### CODE REVIEW

File: android_docker_image.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
+The semaphore `android` docker image is a customized image based on `ubuntu:16.04`
```
Comment (markoa) [inline android_docker_image.md]: Please use capital letter where necessary, Semaphore, Android etc.

Comment (markoa) [reply android_docker_image.md]: This sentence is not correct because there is more than one image for Android (the one with Node for React Native people). So we need a different formulation.

File: android_docker_image.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+optimized for CI/CD. It comes with a set of preinstalled utility tools commonly
+used for CI/CD workflows.
+
+[[__TOC__]]
+
+## Using the semaphoreci/android docker image in your semaphore.yml configuration
```
Comment (markoa) [inline android_docker_image.md]: Rephrase this to something that a stranger to Semaphore can understand.

Eg. Configuring your Android project on Semaphore

File: android_docker_image.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+``` yaml
+version: v1.0
+name: Hello andorid
+agent:
+  machine:
+    type: e1-standard-2
+  containers:
+    - name: main
+      image: semaphoreci/android:28
```
Comment (markoa) [inline android_docker_image.md]: Is it intentional to use 28 and not 29?

File: android_docker_image.md [hunk 4]
```diff
@@ -0,0 +1,109 @@
+The semaphore `android` docker image is a customized image based on `ubuntu:16.04`
+optimized for CI/CD. It comes with a set of preinstalled utility tools commonly
+used for CI/CD workflows.
+
+[[__TOC__]]
+
+## Using the semaphoreci/android docker image in your semaphore.yml configuration
+
+To use the images in this repository for your CI/CD pipelines, set up your Semaphore YAML with:
```
Comment (markoa) [inline android_docker_image.md]: This should explain for what projects this configuration is good for. Kotlin, XYZ...

File: android_docker_image.md [hunk 5]
```diff
@@ -0,0 +1,109 @@
...
+  - name: "Hello"
+    task:
+      jobs:
+      - name: Hello
+        commands:
+          - checkout
+          - echo "Hello"
+```
+
+## Dockerhub repository
```
Comment (markoa) [inline android_docker_image.md]: Ideally this would be the very last section of this doc that only very brave and experienced people will need to pay attention to.

File: android_docker_image.md (hunk 4 above)
Comment (markoa) [reply android_docker_image.md]: This example should be followed by an example for React Native. That should be enough info for 90+% of people.

File: android_docker_image.md [hunk 6]
```diff
@@ -0,0 +1,109 @@
...
+The full list of android docker containers can be found under [SemaphoreCI docker registry](https://hub.docker.com/repository/docker/semaphoreci/android).
+
+## Dockerfiles
+Dockerfiles used to build an android image can be found under [Docker images repository](https://github.com/semaphoreci/docker-images/tree/master/dockerfiles/android).
+
+## Image Variants
+
+The android image.
```
Comment (markoa) [inline android_docker_image.md]: This is just hanging here.

File: android_docker_image.md [hunk 7]
```diff
@@ -0,0 +1,109 @@
...
+
+ - andorid:version is the basic image that comes with development utilities and is an optimal first choice for andorid.
```
Comment (markoa) [inline android_docker_image.md]: Typo, use `code`

Comment (markoa) [reply android_docker_image.md]: Revise this section after you see how the new structure looks as per earlier comments.

File: android_docker_image.md [hunk 8]
```diff
@@ -0,0 +1,109 @@
...
+- build-tools;28.0.0
+- build-tools;27.0.3
+- build-tools;27.0.2
+- build-tools;27.0.1
+- build-tools;27.0.0
+- build-tools;26.0.2
+- build-tools;26.0.1
+- build-tools;25.0.3
+
+### Installed software
```
Comment (markoa) [inline android_docker_image.md]: Sort alphabetically please.

**Comment by CommanderK5** [comment]
@markoa changed the document structure a bit.

**Comment by markoa** [comment]
I've made some further changes, motivated by the upcoming docs freeze to have something out there. :)
